### PR TITLE
Update regions.csv and geojson files for HIP 100

### PR DESCRIPTION
**Pull request by waveform06**
HIP 100 updates
IN865.geojson to remain as is
AS923.geojson to remain as is
Unknown.geojson to include Sudan and Libya
EU433.geojson to include Lesotho
EU868.geojson to remove Sudan, Libya and Lesotho
EU868.geojson to include the Islands of 
SH-AC https://www.google.com/maps/place/Ascension+Island/ SH-HL https://www.google.com/maps/place/Saint+Helena/ SH-TA https://www.google.com/maps/place/Tristan+da+Cunha includes 
https://www.google.com/maps/place/Inaccessible+Island https://www.google.com/maps/place/Nightingale+Islands https://www.google.com/maps/place/Gough+Island

### CODE REVIEW

**Comment by waveform06** [comment]
For the EU868.geojson/Unknown.geojson borders that are not agreed between Sudan and neighbors:
Between Egypt and Sudan:
Use the 22nd parallel of latitude as the border at the https://en.wikipedia.org/wiki/Halaib_Triangle
Between  South Sudan and Sudan:
Use the northern border for the areas of https://en.wikipedia.org/wiki/Abyei and https://en.wikipedia.org/wiki/Kafia_Kingi

**Comment by gradoj** [comment]
Ascension Island, Saint Helena, Tristan da Cunha, Inaccessible, Nightingale and Gough Island are all one group in Unknown with Index -1964272. Just need to move geojson to EU868 from Unknown.

Please confirm whether disputed Abyei and Kafia will be EU868 or Unknown.

**Comment by waveform06** [comment]
Abyei and Kafia will be EU868

File: regions.csv [hunk 1]
```diff
@@ -123,9 +123,9 @@ KG,KGZ,41.20438,74.766098,Kyrgyzstan,Unknown,,,1.0.4,zone1,-178009,
 LA,LAO,19.85627,102.495496,Lao Peoples Democratic Republic,AS923-1,EU868,EU433,1.0.4,zone3,-49903,
 LV,LVA,56.879635,24.603189,Latvia,EU868,EU433,,1.0.4,zone1,-72594,
 LB,LBN,33.854721,35.862285,Lebanon,EU868,EU433,,1.0.4,zone1,-184843,
-LS,LSO,-29.609988,28.233608,Lesotho,EU868,EU433,,,zone1,-2093234,
+LS,LSO,-29.609988,28.233608,Lesotho,EU433,EU868,,,zone1,-2093234,
 LR,LBR,6.428055,-9.429499,Liberia,EU868,,,,zone1,-192780,
-LY,LBY,26.3351,17.228331,Libya,EU868,,,,zone1,-192758,
+LY,LBY,26.3351,17.228331,Libya,Unkownn,,,,zone1,-192758,
```
Comment (jthiller) [inline regions.csv:128]: typo here

**Comment by gradoj** [comment]
EU868
image

**Comment by waveform06** [comment]
Well it looks like there is another obscure border issue with Kenya I missed
https://en.wikipedia.org/wiki/Ilemi_Triangle ought to be EU868 as its now South Sudan and the adjacent bit next to Kokuro
Claimed as https://en.wikipedia.org/wiki/Eastern_Equatoria and Kenya dont seem to claim it on their maps
https://en.wikipedia.org/wiki/Geography_of_Kenya

**Comment by waveform06** [comment]
llemi triangle now has its own issue for later integration
https://github.com/dewi-alliance/hplans/issues/55

**Pull request merged**